### PR TITLE
Dịch "lr-scheduler_vn" - Phần 4 

### LINKED ISSUE #1991: Dịch "lr-scheduler_vn" - Phần 4
Phần cần dịch: https://github.com/aivivn/d2l-vn/blame/master/chapter_optimization/lr-scheduler_vn.md#L233-L315

### Hướng dẫn
- [Dịch trực tiếp trên website](https://github.com/aivivn/d2l-vn/blob/master/CONTRIBUTING.md).
- Sau khi nhấn vào phần cần dịch ở trên, hãy chọn **Normal view** và làm theo hướng dẫn ở trên để có thể bắt đầu dịch.

### PR DESCRIPTION
**Pull request by davidnvq**
## Pull Request Checklist

**Đánh dấu tick (sau khi "Create Pull Request") vào các mục dưới đây:**

* [x] Pull Request này tương ứng với issue nào? Điền số issue sau dấu `#` (không có dấu cách): Close #1991

* [x] Bản dịch này có bám sát **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)** không? Nếu một từ/cụm chưa có trong Bảng thuật ngữ, hãy tạo một Pull Request riêng đề xuất cách dịch từ/cụm đó.

* [x] Format của bản gốc có được giữ nguyên không?
     
* [x] Trong một đoạn văn, mỗi câu văn đã được viết trong một dòng, giữa các dòng không có dòng trắng. Markdown sẽ coi những dòng liền nhau không có dòng trắng là một đoạn văn. Việc này giúp công đoạn review được thuận tiện hơn. Bạn đã kiểm tra rằng mỗi đoạn văn đã nằm một dòng?

* [x] Điền tên đầy đủ của bạn vào mục **Những người thực hiện** ở cuối tập tin.

* [ ] Điền tên những reviewers mà bạn thấy có nhiều đóng góp cho Pull Request này. Xem danh sách Contributors **[tại đây](https://github.com/aivivn/d2l-vn/blob/master/docs/contributors_info.md)**.

## Lưu ý cho người phản biện (Reviewer)

1. Chỉ review nếu Pull Request (PR) này đã sẵn sàng cho việc review. Một PR sẵn sàng khi PR đó không có label `status: wip`.
2. Đã đọc kỹ hướng dẫn đóng góp và xem qua **[Bảng thuật ngữ](https://github.com/aivivn/d2l-vn/blob/master/glossary.md)**.

### CODE REVIEW

**Comment by davidnvq** [comment]
có từ warmup tạm dịch là `làm nóng`, mà thấy `khởi động` cũng đúng. đợi #2008 rồi sửa vậy

File: chapter_optimization/lr-scheduler_vn.md [hunk 1]
```diff
@@ -236,14 +236,15 @@ Lastly, on some problems it is beneficial to warm up the optimizer prior to usin
...
-### *dịch tiêu đề phía trên*
+### Định thời Thừa số
 
 <!--
 One alternative to a polynomial decay would be a multiplicative one, that is $\eta_{t+1} \leftarrow \eta_t \cdot \alpha$ for $\alpha \in (0, 1)$.
 To prevent the learning rate from decaying beyond a reasonable lower bound the update equation is often modified to $\eta_{t+1} \leftarrow \mathop{\mathrm{max}}(\eta_{\mathrm{min}}, \eta_t \cdot \alpha)$.
 -->
 
-*dịch đoạn phía trên*
+Một giải pháp thay thế cho suy giảm đa thức đó là sử dụng thừa số nhân,$\eta_{t+1} \leftarrow \eta_t \cdot \alpha$ với $\alpha \in (0, 1)$.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:246]:
```suggestion
Một giải pháp thay thế cho suy giảm đa thức đó là sử dụng thừa số nhân $\alpha \in (0, 1)$, lúc này $\eta_{t+1} \leftarrow \eta_t \cdot \alpha$.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 2]
```diff
@@ -269,7 +270,10 @@ Going forward we will use the built-in schedulers as appropriate and only explai
 As illustrated, it is fairly straightforward to build your own scheduler if needed.
 -->
 
-*dịch đoạn phía trên*
+Cách khắc phục trên cũng có thể được thực hiện bằng một định thời có sẵn trong MXNet thông qua đối tượng `lr_scheduler.FactorScheduler`.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:273]:
```suggestion
Cách trên cũng có thể được thực hiện bằng một bộ định thời có sẵn trong MXNet `lr_scheduler.FactorScheduler`.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 3]
```diff
@@ -269,7 +270,10 @@ Going forward we will use the built-in schedulers as appropriate and only explai
...
+Giải pháp này cần ít tham số hơn, ví dụ gồm có chu kỳ khởi động (_warmup period_), phương thức khởi động (_warmup mode_), số bước cập nhật tối đa, v.v.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:274]:
```suggestion
Cách này yêu cầu nhiều tham số hơn một chút, ví dụ như thời gian khởi động (_warmup period_), chế độ khởi động (_warmup mode_), số bước cập nhật tối đa, v.v.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 4]
```diff
@@ -269,7 +270,10 @@ Going forward we will use the built-in schedulers as appropriate and only explai
...
+Ở các phần tiếp theo, chúng ta sẽ sử dụng các định thời tốc độ học thích hợp và còn ở đây sẽ giải thích cách thức hoạt động của chúng.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:275]:
```suggestion
Ở các phần tiếp theo, chúng ta sẽ sử dụng các bộ định thời tốc độ học được lập trình sẵn, ở đây chỉ giải thích cách thức hoạt động của chúng.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 5]
```diff
@@ -279,15 +283,17 @@ As illustrated, it is fairly straightforward to build your own scheduler if need
...
 
 <!--
 A common strategy for training deep networks is to keep the learning rate piecewise constant and to decrease it by a given amount every so often.
 That is, given a set of times when to decrease the rate, such as $s = \{5, 10, 20\}$ decrease $\eta_{t+1} \leftarrow \eta_t \cdot \alpha$ whenever $t \in s$.
 Assuming that the values are halved at each step we can implement this as follows.
 -->
 
-*dịch đoạn phía trên*
+Một chiến lược chung để huấn luyện các mạng nơ-ron sâu là giữ cho tốc độ học không đổi theo từng khoảng và thường xuyên giảm tốc độ học đi một lượng cho trước sau mỗi khoảng.
+Cụ thể, với một tập giá trị các bước thời gian từ đó ta giảm tốc độ học, ví dụ như với $s = \{5, 10, 20\}$, ta giảm $\eta_{t+1} \leftarrow \eta_t \cdot \alpha$ khi $t \in s$.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:295]:
```suggestion
Cụ thể, với một tập thời điểm giảm tốc độ học, ví dụ như với $s = \{15, 30\}$, ta giảm $\eta_{t+1} \leftarrow \eta_t \cdot \alpha$ khi $t \in s$.
```
sửa khoảng cho giống code.

File: chapter_optimization/lr-scheduler_vn.md [hunk 6]
```diff
@@ -303,7 +309,9 @@ Then (and only then) do we decrease the rate such as to obtain a higher quality
 The example below shows how this can produce ever slightly better solutions.
 -->
 
-*dịch đoạn phía trên*
+Ý tưởng trực quan đằng sau định thời tốc độ học không đổi theo từng khoảng đó là phương pháp này cho phép quá trình tối ưu xảy ra cho tới khi thuật toán có phân phối của các vector trọng số đạt tới điểm ổn định theo nghĩa.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:312]:
```suggestion
Ý tưởng trực quan đằng sau định thời tốc độ học không đổi theo khoảng đó là phương pháp này cho phép quá trình tối ưu xảy ra cho tới khi thuật toán đạt tới điểm ổn định về phân phối của các vector trọng số.
```

File: chapter_optimization/lr-scheduler_vn.md [hunk 7]
```diff
@@ -303,7 +309,9 @@ Then (and only then) do we decrease the rate such as to obtain a higher quality
...
+Khi và chỉ khi đạt được trạng thái đó, chúng ta mới giảm tốc độ học để đạt được biến đại diện cho điểm cực tiểu chất lượng hơn.
```
Comment (cuongvng) [inline chapter_optimization/lr-scheduler_vn.md:313]:
```suggestion
Khi và chỉ khi đạt được trạng thái đó, chúng ta mới giảm tốc độ học hướng tới điểm cực tiểu chất lượng hơn.
```
